quantum/util/impl/quantum_util_impl.h

```diff
@@ -34,7 +34,8 @@ int bindCoro(Traits::Yield& yield,
     try
     {
         ctx->setYieldHandle(yield); //set coroutine yield
-        yield.get() = std::forward<CAPTURE>(capture)();
+        int rc = std::forward<CAPTURE>(capture)();
```

Comment:
A comment is needed here to explain that the return address must be on the coroutine stack so that it can be properly captured in the coroutine context. Otherwise when multiple threads run the same coroutine, the return value will be copied in the wrong location.

Reply:
sure will add. I am curious if it's ever possible that the compiler will optimize this local variable out